cmd/runsim/integrations/slack/main.go

```diff
@@ -19,95 +19,54 @@ import (
 
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/cosmos/tools/lib/runsimaws"
+	"github.com/cosmos/tools/lib/runsimslack"
 )
 
-const paramNameSlackSecret = "slack_signing_secret"
-const paramNameCircleToken = "circle_build_token_simulation"
+const (
+	// Security token IDs
+	ssmSlackSecretId   = "slack-cmd-secret"
+	ssmCircleTokenId   = "circle-token-sim"
+	ssmSlackChannelId  = "slack-channel-id"
+	ssmSlackAppTokenId = "slack-app-key"
 
-var (
-	err      error
-	response *http.Response
-	request  *http.Request
-)
-
-type CircleStatusCheckResp struct {
-	Status    string `json:"status"`
-	Lifecycle string `json:"lifecycle"`
-}
+	slashCmd = "/sim_start"
+	slashCmdDev = "/dev_sim_start"
 
-type CircleJobTriggerResp struct {
-	Status   string `json:"status"`
-	Body     string `json:"body"`
-	BuildURL string `json:"build_url"`
-}
+	// DynamoDB attribute and table names
+	awsRegion = "us-east-1"
+	simStateTable = "SimulationState" // name of the dynamodb table where details from running simulations are stored
+	primaryKey    = "IntegrationType" // primary partition key used by the sim state table
+)
 
 type CircleApiPayload struct {
```

Comment:
we can move all these common types into a `cmd/runsim/integrations/common` package

Edit: or just use the one from the github integration